server/src/test/java/org/elasticsearch/action/bulk/TransportBulkActionTests.java

```diff
@@ -132,4 +160,55 @@ public void testDeleteNonExistingDocExternalGteVersionCreatesIndex() throws Exce
             throw new AssertionError(exception);
         }));
     }
+
+    public void testRoutingResolvesToWriteIndexAliasMetaData() throws Exception {
```

Comment:
I wonder if we need a test at this level. We already unit test MetaData and we have an IT test to check it works end to end.

Reply:
actually, I don't think we have an IT test, but we should

Reply:
I couldn't find an IT test that verifies this, and I thought that this level of unit test was sufficient. Will add IT test (will also double check that one doesn't exist)

Reply:
I think this can be removed now that we have an IT test? this layer doesn't add anything to the write index resolving as it is all in the resolver? it's just glue and is covered well by the IT?